src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -22,6 +22,8 @@ import { CellTemplate } from '~/app/shared/enum/cell-template.enum';
 import { CdTableColumn } from '~/app/shared/models/cd-table-column';
 import { CdTableFetchDataContext } from '~/app/shared/models/cd-table-fetch-data-context';
 import { Daemon } from '~/app/shared/models/daemon.interface';
+import { CephServiceSpec } from '~/app/shared/models/service.interface';
+import { ListToHtmlPipe } from '~/app/shared/pipes/list-to-html.pipe';
 import { RelativeDatePipe } from '~/app/shared/pipes/relative-date.pipe';
 
 @Component({
```

Comment:
In the @Component section you should add the `ChangeDetectionStrategy.OnPush` (and retry with the ng-profiler to see how much this varies):

```suggestion
import { ListToHtmlPipe } from '~/app/shared/pipes/list-to-html.pipe';
import { RelativeDatePipe } from '~/app/shared/pipes/relative-date.pipe';

@Component({
  changeDetection: ChangeDetectionStrategy.OnPush,
```

Reply:
Before the change: Object { msPerTick: 1.0570824524312896, numTicks: 473 }
After: After: Object { msPerTick: 0.8695652173913043, numTicks: 575 }